documentation/src/main/asciidoc/getting-started.asciidoc

```diff
@@ -441,6 +441,13 @@ include::{sourcedir}/org/hibernate/search/documentation/gettingstarted/withhsear
 ----
 ====
 
+=== Showcase Project
+
+The project [Hibernate Search with Hibernate ORM Showcase](https://github.com/hibernate/hibernate-search/tree/master/integrationtest/showcase/library)
```

Comment:
This feels a bit... out of place? Maybe this would be better at the end of the "What's next" section below.

Also, rephrased like this, it would feel more natural:

```
If you want to see an example project using Hibernate Search,
you can also have a look at the ["Library" showcase](https://github.com/hibernate/hibernate-search/tree/master/integrationtest/showcase/library),
a sample application using Hibernate Search in a Spring Boot environment.
```